### PR TITLE
Add design doc for one-round serial queuer

### PR DESCRIPTION
**Pull request by CLEckhardt**
Add details of algorithm and implementation of one-round serial queuing
strategy.

Signed-off-by: Chris Eckhardt <eckhardt@bitwise.io>

### CODE REVIEW

File: community/planning/queuer_strat_one_round_serial.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# One-round serial queuer strategy
```
Comment (shannynalayna) [inline community/planning/queuer_strat_one_round_serial.md:1]: Need to add the copyright to this file

Comment (shannynalayna) [reply community/planning/queuer_strat_one_round_serial.md:1]: Also need to add a link to this doc somewhere, it's inaccessible atm

Comment (CLEckhardt) [reply community/planning/queuer_strat_one_round_serial.md:6]: Fixed, thanks!

File: community/planning/queuer_strat_one_round_serial.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+<!--
+  Copyright 2022 Cargill Incorporated
```
Comment (Caleb-Hill) [inline community/planning/queuer_strat_one_round_serial.md:2]: I think this header should say 2018-2022 instead of just 2022

Comment (CLEckhardt) [reply community/planning/queuer_strat_one_round_serial.md]: updated, thanks!

**Review by ameliabradley — APPROVED** [review]

**Pull request merged**